docs/api.md

```diff
@@ -167,11 +167,12 @@ be used in combination with the `useContext` hook.
 The client manages all operations and ongoing requests to the exchange pipeline.
 It accepts a bunch of inputs when it's created
 
-| Input        | Type                               | Description                                                                                  |
-| ------------ | ---------------------------------- | -------------------------------------------------------------------------------------------- |
-| url          | `string`                           | The GraphQL API URL as used by `fetchExchange`                                               |
-| fetchOptions | `RequestInit \| () => RequestInit` | Additional `fetchOptions` that `fetch` in `fetchExchange` should use to make a request       |
-| exchanges    | `Exchange[]`                       | An array of `Exchange`s that the client should use instead of the list of `defaultExchanges` |
+| Input        | Type                               | Description                                                                                                     |
+| ------------ | ---------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| url          | `string`                           | The GraphQL API URL as used by `fetchExchange`                                                                  |
+| fetchOptions | `RequestInit \| () => RequestInit` | Additional `fetchOptions` that `fetch` in `fetchExchange` should use to make a request                          |
+| suspense     | `?boolean`                         | Activates the experimental React suspense mode, which can be used during server-side rendering to prefetch data |
```

Comment:
Maybe worth noting here (if its the case) that this doesn't work with client-side suspense.

Reply:
I haven't tested it but it should actually work so I didn't want to rule it out here 😅